images/ose-k8s-prometheus-adapter.yml

```diff
@@ -10,7 +10,7 @@ from:
   builder:
   - image: openshift/golang-builder:1.10
   member: openshift-enterprise-base
-name: openshift/ose-prometheus-adapter
+name: openshift/ose-k8s-prometheus-adapter
```

Comment:
the image name change is fine, but I don't think you're changing the dist-git (`containers/ose-k8s-prometheus-adapter` does not exist) so you should also not change the file name.

Reply:
got it, I wasn't sure if that's necessary or not .. will revert the filename change